indico/modules/designer/templates/_list.html

```diff
@@ -60,31 +58,55 @@
         <td>
             <div class="thin toolbar right">
                 <div class="group">
-                    {% if tpl.is_clonable %}
-                        <button class="i-button icon-copy icon-only text-color borderless hide-if-locked"
+                    {% if not event and tpl.type.name == 'badge' %}
+                        {% set inherited_from_root = default_template == tpl and tpl.category.get_root().default_ticket_template == tpl %}
+                        <a class="i-button icon-ticket icon-only borderless hide-if-locked toggle-default
+                            {{ 'default-on-category' if default_template == tpl }}
+                            {{ 'inherited-from-root' if inherited_from_root }}"
+                            {% if inherited_from_root %}
+                                title="{% trans %}This is the default ticket template from Home category{% endtrans %}"
+                            {% else %}
+                                title="{% trans %}Toggle ticket template for this category{% endtrans %}"
+                                data-href="{{ url_for('.toggle_category_default', target, template_id=tpl.id) }}"
+                                data-method="POST"
+                                data-update="#template-list"
+                            {% endif %}>
+                        </a>
+                    {% endif %}
+                        {% set clonable = tpl.is_clonable or target == tpl.category.get_root() %}
```

Comment:
But its to check whether the page we display this list is a root category page, thats the `target`object but it can also be an event,  or should I check `target.is_root` even though it can be an Event object ?